src/main/java/io/gatling/mojo/EnterpriseStartMojo.java

```diff
@@ -110,6 +125,21 @@ public final class EnterpriseStartMojo extends AbstractEnterprisePluginMojo {
   @Parameter(property = "gatling.enterprise.simulationEnvironmentVariables")
   private String simulationEnvironmentVariablesString;
 
+  /**
+   * Defines the behavior of the {@code enterpriseStart} goal.
+   *
+   * <ul>
+   *   <li><b>FIRE_AND_FORGET:</b> Do not wait for the result after starting the simulation on
+   *       Gatling Enterprise.
+   *   <li><b>WAIT_FOR_RUN_END:</b> Wait for the result after starting the simulation on Gatling
```

Comment:
Was it the use case for this option?

Reply:
We included this possibility in the GitHub Action, and the intention of this feature was to make it easier to run from a CI system without a dedicated plugin by simply using the enterpriseStart command instead, so I just replicated the same possibility. But I don't actually remember why we included it in the GitHub Action the first place...

(On the GitHub action, it's handled by the `fail_action_on_run_failure` boolean parameter.)

Reply:
As discussed, we only need a boolean fire-and-forget/blocking and always fail whatever the error (crash on boot, crash during the run, assertion failure). We'll iterate from ths.

Reply:
We discussed this, will just have a boolean option and always fail on any error. We'll see later if and how to handle specific use cases around this.